assets/js/edit.js

```diff
@@ -436,7 +436,18 @@ function Edit( props ) {
 				{src ? (
 					<>
 						<div className="wp-block-podcasting-podcast__container">
-							{showImage && displayArt && (
+							{displayArt && featuredImageUrl && (
+								<div className="wp-block-podcasting-podcast__show-art">
+									<div className="wp-block-podcasting-podcast__image">
+										<img
+											src={featuredImageUrl}
+											alt={showName}
+										/>
+									</div>
+								</div>
+							)}
+
+							{displayArt && !featuredImageUrl && showImage && (
```

Comment:
If possible, would be great to combine all of this in a single conditional, as the markup we render is the same for both and right now we have pretty much duplicate code. Something along the lines of check if `displayArt` is `true` and then within that conditional, determine if we are showing the featured image or podcast image

Reply:
This is handled in https://github.com/10up/simple-podcasting/pull/318/commits/33d39ae6aed0e94a612346c021b0f355443d6d07